UnitySDK/Assets/PlayerWon-SDK/Classes/ClientDetails.cs

```diff
@@ -61,8 +61,24 @@ public class ClientDetails
         [SerializeField]
         private int gs;
 
-        private ClientDetails(string _country, string _idfa, string _plat, string _lang, string _ip = null, int _w = 0, int _h = 0, string _pid = null, string _pidType = null, string _device = null, int _coppa = 0,
-            int _underage = 0, int _age = 0, string _slot_id = null, int _ls = 0, Currency _lsv = null, CurrencyArray _rv = null, int _gs = 0)
+        [SerializeField]
+        private int lt;
```

Comment:
nit but might affect how this is used. If this field is private, the only way to set it is through the editor in edit mode. If you make these fields public, one could set them through code as well. Some fields that might change at runtime (e.g. lsv, lt, etc) that might be more desirable

Reply:
ps: hello :)

Reply:
Intruder alert!